kpops/components/streams_bootstrap/base.py

```diff
@@ -81,3 +85,21 @@ def warning_for_latest_image_tag(self) -> Self:
                 f"The image tag for component '{self.name}' is set or defaulted to 'latest'. Please, consider providing a stable image tag."
             )
         return self
+
+    @override
+    def manifest_deploy(self) -> tuple[KubernetesManifest, ...]:
+        resource = super().manifest_deploy()
+        if self.to:
+            resource = resource + self.manifest_strimzi_topics(self.to.kafka_topics)
```

Comment:
no need to pass the argument to an instance method
```suggestion
            resource = resource + self.manifest_strimzi_topics()
```

Reply:
The reason I did this was because of pyright. If I do the None check outside and use `self.to.kafka_topic` in the method it will throw me the error: 
```
error: "kafka_topics" is not a known member of "None" (reportOptionalMemberAccess)
```